capabilities/documentation-quality.md

```diff
@@ -1,51 +1,54 @@
 
-Next, consider the descriptions below. Don't worry if the description doesn't exactly match your situation. These descriptions are meant to be examples of situations that would qualify for the associated score.
+Don't worry if the description doesn't exactly match your situation. These descriptions are meant to be examples of situations that would qualify for the associated score.
 
 1. **Minimal:** The technical documentation is often outdated, incomplete, or inaccurate, making it difficult to rely on when working with the services or applications. It's hard to find what is needed, and others are often asked for help.
 2. **Basic:** The technical documentation is somewhat reliable, but it's not always easy to find what is needed. Updates are sporadic, and multiple sources must be dug through to get the required information. In times of crisis, the documentation might be glanced at, but it's not always trusted.
 3. **Good:** The technical documentation is generally reliable, and what is needed can usually be found with some effort. Updates are made regularly, but not always immediately. The documentation is used to help troubleshoot issues, but clarification from others might still be needed.
 4. **Excellent:** The technical documentation is comprehensive, accurate, and up-to-date. What is needed can easily be found, and the documentation is relied on heavily when working with the services or applications. When issues arise, the documentation is confidently reached for to help troubleshoot and resolve problems.
 
-The number you selected represents your overall score for this capability. If you feel like your team or organization fits somewhere in-between two scores, it's okay to use a decimal. For example, if you think your team or organization has somewhere between basic and good Documentation Quality, you would score a 2.5.
+The number you selected represents your overall score for this capability. If you feel like your team or organization fits somewhere in between two scores, it's okay to use a decimal. For example, if you think your team or organization has somewhere between basic and good Documentation Quality, you would score a 2.5.
 
 An overall score of 2.5 or below means you'll probably gain a lot of value from experimenting with some of the supporting practices listed below. An overall score of 3 or higher means you and your team are largely proficient, or well on your way to becoming proficient, in the area of Documentation Quality; you would likely benefit from evaluating your scores in other capabilities.
 
 ## Supporting Practices
```

Comment:
Add to this list the Practice of "Check Documentation Consistency"? If so, I can add the following (taken/altered from Version Control capability):

### [Check Documentation Consistency](/practices/check-documentation-consistency.md)
Checking documentation consistency ensures that all documentation is kept up-to-date and consistent with the codebase it describes. This practice facilitates better alignment between documentation and software development, ensuring that changes in code are accurately reflected in the documentation, thereby enhancing the overall quality and reliability of both the codebase and the supporting documents.

Reply:
Great catch! That seems like a duplicate of `Schedule Regular Documentation Audits` which is already here. I like the idea of doing this activity as a routine instead of a one-off. So maybe we replace `Check Documentation Consistency` and favor `Schedule Regular Documentation Audits`?? Feel free to pull content over from `Check Documentation Consistency` if you see some valuable stuff there.

Reply:
That makes sense to me. I'll replace Check Documentation Consistency with Schedule Regular Documentation Audits in Version Control. I'll also pull and merge any valuable content from Check Documentation Consistency here (in Schedule Regular Documentation).

Reply:
Done in 11/14 commit.